### ISSUE TITLE
Should we report col offset in some tokenizer errors?

### ISSUE THREAD

**Issue by pablogsal** [we-like-parsers/cpython#110]
```
./python 
Python 3.9.0a5+ (heads/more_errors:54e6bc98a7, Apr 21 2020, 03:55:06) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> def f():
...     pass
...   pass
  File "<stdin>", line 3
IndentationError: unindent does not match any outer indentation level
```

```
╰─ ./python -X oldparser
Python 3.9.0a5+ (heads/more_errors:54e6bc98a7, Apr 21 2020, 03:55:06) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> def f():
...     pass
...   pass
  File "<stdin>", line 3
    pass
       ^
IndentationError: unindent does not match any outer indentation level
```

_Originally posted by @lysnikolaou in https://github.com/we-like-parsers/cpython/pull/105#issuecomment-616886748_

**Comment by ammaraskar** [comment]
Let's merge this with #121 which is being worked on by @isidentical

**Comment by lysnikolaou** [comment]
Yup, this will probably be resolved once the fix for #121 is merged.

**Comment by lysnikolaou** [comment]
Fixed in python/cpython#19782.
